### ISSUE TITLE
Wrong allowed IPs in wiki

### ISSUE THREAD

**Issue by MrArtemSid** [xtrime-ru/antizapret-vpn-docker#78]
https://github.com/xtrime-ru/antizapret-vpn-docker/blob/master/docs/guide_Keenetic.md?plain=1#L101

0.0.0.0.0 instead of 0.0.0.0

**Comment by Dimariqe** [comment]
Thank you, I will send the changes soon

**Comment by Dimariqe** [comment]
I agree, inexperienced users with this instruction often encounter a problem when connecting another device all traffic goes through VPN. I'm thinking of making a TRICKS section for experienced users, where I'll add some recipes.

**Comment by xtrime-ru** [comment]
I think, experienced users do not read docs ;)
